client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel/incident-info-panel.component.ts

```diff
@@ -62,6 +62,12 @@ export class IncidentInfoPanel implements AfterViewInit {
     else return 'A wildfire of undetermined cause, including a wildfire that is currently under investigation, as well as one where the investigation has been completed.'
   }
 
+  public getResponseTypeDescription (code: string) {
+    if (code === 'MONITOR') return 'When a fire is being monitored, this means BC Wildfire Service is observing and analyzing the fire but it\'s not immediately suppressed. It may be allowed to burn to achieve ecological or resource management objectives and is used on remote fires that do not threaten values.'
```

Comment:
Just a typo here for "it's" @dhlevi

Reply:
Not a Typo! the slash is needed to break the single-quote. I could change the string to double-quotes but that would be inconsistent.